src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -60,11 +79,25 @@ public static String buildSearchConceptsQuery(
     BooleanAndOrFilterVariable whereClause =
         new BooleanAndOrFilterVariable(BooleanAndOrFilterVariable.LogicalOperator.AND, allFilters);
 
-    // select nameField, idField from conceptTable WHERE
-    // domainClause AND (searchNameClause OR searchCodeClause)
     // TODO: DC-845 Implement pagination, remove hardcoded limit
+    // SELECT concept_name, concept_id, COUNT(DISTINCT person_id) as count
+    // FROM concept JOIN domain_occurrence ON domain_occurrence.concept_id =
+    // concept.concept_id
+    // WHERE concept.name CONTAINS {{name}} GROUP BY c.name, c.concept_id
```

Comment:
Could you add the order by clause to the comment too?

Reply:
Yep! I agree with you this will make things easier for those reading. I do think that in the future, the query builders need stronger comment coverage for reusability purposes.